composer.json

```diff
@@ -4,7 +4,7 @@
   "require": {
     "php": ">=5.4.0",
     "laravel/framework": "~5.0",
-    "laravel/socialite": "~2.0"
+    "laravel/socialite": "~3.0"
```

Comment:
Version should be **3.0.*@dev** because Socialite 3.0 isn't tagged as stable as of now.